config.json

```diff
@@ -14,7 +14,8 @@
           "craig": "5a09cedd03ac8f6625afbf82320d9182ab56e507311f43d02aace8f5fe6d93f5",
           "igor": "56d20f3d01cf0c08db466fb629187ff41a30bb170623251b7e14e32c3c1a67a4",
           "vratislav": "54a458f06da35064562b258e52f5b233dbbd561a51816097b7bce1a306775d05",
-          "ole": "94534ae37a66ade3bae446c81cc15433efcda3a8516ffe6b2c2e0f6dd70127dd"
+          "ole": "94534ae37a66ade3bae446c81cc15433efcda3a8516ffe6b2c2e0f6dd70127dd",
+          "ole1": "2c04264ee57ffe7708855bccdf11d7c0e5f797c7556cf0b7b710fffdc5e8a925"
```

Comment:
Did you create the hash using:

```
$ echo -n "my-email@gmail.com" | shasum -a 256
```

**Note** the `-n` option to echo to leave off the newline.

( I don't know your email, so I don't know if you did it correctly, just figured I'd check ).